platform/src/main/java/org/stellar/anchor/platform/component/custody/FireblocksBeans.java

```diff
@@ -0,0 +1,32 @@
+package org.stellar.anchor.platform.component.custody;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.stellar.anchor.config.SecretConfig;
+import org.stellar.anchor.platform.config.FireblocksConfig;
+import org.stellar.anchor.platform.custody.service.FireblocksEventsService;
+import org.stellar.anchor.platform.job.FireblocksTransactionsReconciliationJob;
+
+@Configuration
+@ConditionalOnProperty(value = "custody.type", havingValue = "fireblocks")
+public class FireblocksBeans {
+  @Bean
+  @ConfigurationProperties(prefix = "custody.fireblocks")
+  FireblocksConfig fireblocksConfig(SecretConfig secretConfig) {
+    return new FireblocksConfig(secretConfig);
+  }
+
+  @Bean
+  FireblocksTransactionsReconciliationJob reconciliationJob() {
+    return new FireblocksTransactionsReconciliationJob();
+  }
+
+  @Bean
+  FireblocksEventsService fireblocksEventsService(
+      @Value("${custody.fireblocks.public_key}") String publicKey) {
```

Comment:
It's better to use FireblocksConfig here instead of `@Value`
You can reference public key via `cfg.getPuiblicKey()`